pkg/options/network/create.go

```diff
@@ -87,6 +91,9 @@ func BuildNetworkCommandWithOptions() (*cobra.Command, *NetworkUserOptions) {
 		defaultTemplateDir,
 		"Use a specific template directory, overriding the OCM_TEMPLATE_DIR environment variable.",
 	)
+	if exportedTempDir != "" {
+		options.TemplateDir = exportedTempDir
```

Comment:
Not sure I understand this change,

I see this is supposed to set user-inputted `template-dir` to it's original value after the flag is created, but I feel like it would be the same outcome if we changes L91's default value to be `options.templateDir`

Where does the env var feed in? should we be using that as the default value here on L91, and maybe disabling auto mode input later for `template-dir`?

Reply:
Personally I'd be all for replacing L91 , but i do see why it might be confusing to see the default being the path set by the OCM_TEMPLATE_DIR env var

Reply:
unless the env var is set to the `options.templateDir` var prior to this, then it makes sense

Reply:
nm yeah I see that, LGTM